doc/amazon_sagemaker_operators_for_kubernetes.rst

```diff
@@ -82,12 +82,12 @@ IAM role-based operator deployment
 Before you can deploy your operator using an IAM role, associate an OpenID Connect (OIDC) provider with your role to
 authenticate with the IAM service.
 
-Associate an OpenID Connect Provider to Your Instance
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Create an OpenID Connect Provider for Your Cluster
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Create an OIDC identity provider for your cluster. If your
-cluster is managed by EKS, then your cluster will already have an OIDC
-attached to it. 
+Create an OIDC identity provider for your cluster.
+The following command will create and associate an OIDC provider
```

Comment:
I might say "the following instructions" rather than the following command, as what comes after isn't directly a command.